source/fundamentals/connection/connect.txt

```diff
@@ -95,6 +95,14 @@ verify that the connection is successful:
 .. literalinclude:: /code-snippets/connection/stable-api-connect.js
    :language: javascript
 
+.. note::
+
+   As of v4.7, the {+driver-short+} will automatically call the ``MongoClient.connect()``
+   method when using the client to interact with your MongoDB instance, **except** when
+   you use the client to call bulk operations. We recommend manually calling
+   ``MongoClient.connect()`` if you want granular control and insight over
+   your application's connections.
```

Comment:
S: i think it would be useful to be more specific here; "granular control and insight" is correct, but might not be helpful to a newer user. (you could also add info about bulk writes here)

```suggestion
   you use the client to call bulk operations. Call the ``MongoClient.connect()`` method
   explicitly if you are performing bulk operations or if you want to verify that the
   connection was successful.
```